learningresources/migrations/0004_auto_adds_audit_fields.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import datetime
+
+from django.db import models, migrations
+from django.utils.timezone import utc
+
+# pylint: skip-file
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('learningresources', '0003_add_permissions'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='course',
+            name='date_created',
```

Comment:
I don't believe so, only because it's being added by the abstract base class, not by modifying this model directly.